k8s/charts/openebs/README.md

```diff
@@ -1,27 +1,35 @@
+# OpenEBS
+
+**It is based on Helm community chart [openebs](https://github.com/openebs/openebs/tree/master/k8s/charts/openebs)**
+
+[OpenEBS](http://openebs.io/) is a cloud-native storage solution built with the goal of providing containerized storage for containers. Using OpenEBS, a developer can seamlessly get persistent storage for stateful applications on Kubernetes with ease.
+
+## Introduction
+
+This chart bootstraps a [OpenEBS](https://github.com/openebs/openebs) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
 ## Prerequisites
 - Kubernetes 1.7.5+ with RBAC enabled
 - iSCSI PV support in the underlying infrastructure
 
-## Installing OpenEBS 
-```
-helm repo add openebs-charts https://openebs.github.io/charts/
-helm repo update
-helm install openebs-charts/openebs
-```
+## Installing the Chart 
 
-## Installing OpenEBS from Chart codebase
-```
-git clone https://github.com/openebs/openebs.git
-cd openebs/k8s/charts/openebs/
-helm install --name openebs .
+The command deploys OpenEBS on the Kubernetes cluster with the release name `openebs` and the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation:
+
+```bash
+$ helm install tc/openebs --name openebs --namespace=openebs
```

Comment:
Is there a step required to add the repo "tc/openebs" ?

Reply:
Readme changes were meant for partner's private repo. Updating my PR